graphql.gemspec

```diff
@@ -5,7 +5,7 @@ require "graphql/version"
 Gem::Specification.new do |s|
   s.name        = "graphql"
   s.version     = GraphQL::VERSION
-  s.date        = Date.today.to_s
+  s.date        = "2016-10-12"
```

Comment:
@rmosolgo, I'm not sure why you needed this. I never got this issue, [and I just tried without](https://github.com/rmosolgo/graphql-ruby/pull/306/commits/0d5e1a7a6f2647c7c19296a672c829d7e27e10e2), looks like [it built fine](https://travis-ci.org/rmosolgo/graphql-ruby/builds/168083322)...
Did you see that locally or on Travis CI?